salt/cloud/clouds/linode.py

```diff
@@ -993,6 +993,58 @@ def get_password(vm_):
     )
 
 
+def decode_linode_plan_label(label):
```

Comment:
this should probably be a private functions, `_decode_linode_plan_label`

Reply:
good call 👍